apps/storybook/.storybook/main.ts

```diff
@@ -2,25 +2,19 @@ import { type StorybookConfig } from '@storybook/react-vite';
 import remarkGfm from 'remark-gfm';
 
 const config: StorybookConfig = {
-  stories: ['../src/**/*.mdx', '../src/**/*.stories.tsx'],
   framework: '@storybook/react-vite',
+  stories: ['../src/**/*.mdx', '../src/**/*.stories.tsx'],
   addons: [
-    {
-      name: '@storybook/addon-essentials',
```

Comment:
All the essential addons are now part of Storybook core except for `addon-docs`, but since we are already using `addon-docs` separately, it doesn't change much.